plugins/base/src/main/kotlin/signatures/KotlinSignatureProvider.kt

```diff
@@ -220,14 +229,23 @@ class KotlinSignatureProvider(ctcc: CommentsToContentConverter, logger: DokkaLog
                     text(".")
                 }
                 link(p.name, p.dri)
-                text(": ")
+                text(": ", styles = mainStyles + TokenStyle.Operator)
                 signatureForProjection(p.type)
                 p.extra[DefaultValue]?.run {
-                    text(" = $value")
+                    text(" = ", styles = mainStyles + TokenStyle.Operator)
+                    highlightValue(value)
                 }
             }
         }
 
+    private fun PageContentBuilder.DocumentableContentBuilder.highlightValue(v: String) {
+        if(v.matches(HighlightingRegex.isNumber)) text(v, styles = mainStyles + TokenStyle.Number)
```

Comment:
I think this information probably can be obtained from analysis, for me current regex-based solution is unacceptable.

Reply:
This information is absent in Documentable Model for default values because they are stored in string format.

Reply:
Yes, but nothing prevents you from adding types to it. For example a default value for:
```kotlin
fun simpleFun(int: Int = 1, string: String = "string"): String = ""
```
is stored as `KtConstantExpression`. I am not sure how to properly get the type from that as the only place i see is a debug name but i'll try to find it out tomorrow / on monday

Reply:
Yes, of course, I was afraid to change the model that the plugin developers use.
As it's stored as `KtConstantExpression` (I mean an expression), the highlighting becomes harder. 
```kotlin
 fun simpleFun(string: String = "string" + "string2" )
```
It needs not only the type.
Or we can highlight the default values as `token constant`.

Reply:
Okey, i've debugged a couple of cases and i think that for now the best solution would be to recognise only constants (like in my first snippet). If default value has an expression then we will render it as text and not parse its arguments. 

This should cover most of the cases and we can add a proper parser later as this is not that trivial.